src/lib/components/inbox/conversations/composer.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from "react"
+
+import { Keyboard, KeyboardAvoidingView, TextInput, TouchableWithoutFeedback } from "react-native"
+
+import styled from "styled-components/native"
+import colors from "../../../../data/colors"
+import fonts from "../../../../data/fonts"
+
+const Container = styled.View`
+  flexDirection: row
+  justifyContent: space-between
+  alignItems: center
+  borderWidth: 1
+  borderColor: ${colors["gray-regular"]}
+  borderRadius: 3
+  marginBottom: 20
+`
+
+const SendButton = styled.Text`
+  fontFamily: ${fonts["avant-garde-regular"]}
+  fontSize: 12
+  marginRight: 10
+`
+
+interface Props {
+  onSubmit: (text: string) => any,
+}
+
+interface State {
+  active: boolean,
+  text: string,
+}
+
+export default class Composer extends React.Component<any, State> {
```

Comment:
the `Props` interface is not being used in the Component definition, should be `React.Component<Props, State>`